changelogs/unreleased/468-harshthakur9030

```diff
@@ -0,0 +1 @@
+fix: Make API service's ip address and port configurable 
```

Comment:
This changelog file can be removed. Since we introduced the feature in this release, a separate changelog for the fix is not required.